frontend/app/src/theme/community/solarizeddark.ts

```diff
@@ -0,0 +1,66 @@
+import type { Theme } from "../types";
+
+const veryDarkCyan = "#002b36";
+const darkCyan = "#073642";
+const darkCyan2 = "#0c586ba6";
+const darkBlue = "#186899";
+const gray = "#8b9898";
+const lightishGray = "#90aba7";
+const lightGray = "#d6dddc";
+const olive = "#748c09";
+
+const txt = lightGray;
+const txtLight = lightishGray;
+
+export function getTheme(base: Theme): Theme {
+    base.name = "solarizeddark";
+    base.label = "Solarized Dark";
+    base.burst = false;
+    base.bg = veryDarkCyan;
+    base.bd = gray;
+    base.txt = txt;
+    base["txt-light"] = txtLight;
+    base.icon.txt = olive;
+    base.icon.selected = olive;
+    base.button.bg = darkBlue;
+    base.button.hv = olive;
+    base.button.txt = txtLight;

```

Comment:
To my eyes, the `txtLight` looks a bit clashy with the blue button and actually I reckon `txt` looks a bit nicer.